### PR TITLE
refactored per garths instructions, added rule file for html hint

### PR DESCRIPTION
**Pull request by sterlingann**
I think its ok now.  Tested with htmlhint as well as the google structured data testing tool.  I added a ruleset file.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -20,10 +20,16 @@
...
+          <section class="hero-footer__section" itemscope itemtype="http://schema.org/SoftwareApplication" >
+              <meta itemprop="ApplicationCategory"  content="hybrid mobile applications, web development"/>
+              <meta itemprop="OperatingSystem" content="iOS, Android, Windows, Blackberry"/>
+              <meta itemprop="name" content ="Phonegap"/>
+              <meta itemprop="isBasedOn" content ="Apache Cordova" />
+              <meta itemprop="License" content="https://github.com/phonegap/phonegap.github.io/blob/master/LICENSE"/>
             <h2>Create your app with PhoneGap</h2>
             <p>Reuse existing web development skills to quickly make hybrid applications built with HTML, CSS and JavaScript. Create experiences for multiple platforms with a single codebase so you can reach your audience no matter their device.</p>
             <a href="http://docs.phonegap.com" class="button--pill--next">Guides &amp; Docs</a>
+          </div>
```
Comment (GarthDB) [inline index.html]: Seems like an extra `</div>` closing tag.

File: index.html [hunk 2]
```diff
@@ -20,10 +20,16 @@
       <footer>
         <!-- {% include banner.html %}  -->
         <div class="bounds">
-          <section class="hero-footer__section">
+          <section class="hero-footer__section" itemscope itemtype="http://schema.org/SoftwareApplication" >
+              <meta itemprop="ApplicationCategory"  content="hybrid mobile applications, web development"/>
```
Comment (GarthDB) [inline index.html]: Fix the indent of the meta tags - should just be 2 spaces more than the `<section` tag above it.

File: index.html [hunk 3]
```diff
@@ -82,7 +88,11 @@ <h4 class="home-gid__item__title">PhoneGap Build</h4>
               </a>
             </li>
 
-            <li class="home-gid__item">
+            <li class="home-gid__item" itemscope itemtype="http://schema.org/SoftwareApplication">
+            <meta itemprop="ApplicationCategory" content ="hybrid mobile applications"/>
```
Comment (GarthDB) [inline index.html]: indent these `meta` 2 more spaces.

File: index.html [hunk 4]
```diff
@@ -95,7 +105,11 @@ <h4 class="home-gid__item__title">Plugin Library</h4>
               </a>
             </li>
 
-            <li class="home-gid__item">
+            <li class="home-gid__item" itemscope itemtype="http://schema.org/SoftwareApplication"/>
+            <meta itemprop="ApplicationCategory" content ="hybrid mobile applications"/>
```
Comment (GarthDB) [inline index.html]: indent these `meta` tags 2 more spaces.

File: index.html [hunk 5]
```diff
@@ -95,7 +105,11 @@ <h4 class="home-gid__item__title">Plugin Library</h4>
               </a>
             </li>
 
-            <li class="home-gid__item">
+            <li class="home-gid__item" itemscope itemtype="http://schema.org/SoftwareApplication"/>
```
Comment (GarthDB) [inline index.html]: This shouldn't be self closing. It should just be:

```html
<li class="home-gid__item" itemscope itemtype="http://schema.org/SoftwareApplication">
```

File: index.html (hunk 2 above)
Comment (sterlingann) [inline index.html]: Thanks @GarthDB !

File: index.html (hunk 1 above)
Comment (sterlingann) [inline index.html]: im surprised that escaped htmlhint.  thanks!

File: index.html (hunk 5 above)
Comment (sterlingann) [inline index.html]: makes sense.  closed list at bottom of the entries with a </li>

File: index.html [hunk 6]
```diff
@@ -20,7 +20,12 @@
       <footer>
         <!-- {% include banner.html %}  -->
         <div class="bounds">
-          <section class="hero-footer__section">
+          <section class="hero-footer__section" itemscope itemtype="http://schema.org/SoftwareApplication" >
+                <meta itemprop="ApplicationCategory"  content="hybrid mobile applications, web development"/>
```
Comment (GarthDB) [inline index.html]: These indents still need addressing; they should be just inside the parent `section` and at the same level with the following `h2` tag.

File: index.html [hunk 7]
```diff
@@ -106,7 +119,7 @@ <h4 class="home-gid__item__title">Third Party Tools</h4>
                   View the tools
                 </footer>
               </a>
-            </li>
+              </li>
```
Comment (GarthDB) [inline index.html]: this closing tag should be at the same indent as the opening `<li` tag (minus 2 space)

**Pull request merged**